### ISSUE TITLE
Possibility of using fzf action + shell commands on the --bind flag?

### ISSUE THREAD

**Issue by nordmagnus999** [junegunn/fzf#2141]
- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

## Info

- OS
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash
    - [ ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

I know how to chain/use multiple shell commands on the `--bind` flag, but i'm wondering if there a way to do this for fzf action + shell command combination too? Example:

```echo '' | fzf --bind:"alt-q:abort ; echo 'hello'"```

something along that? just curious to know.

**Comment by dsimon28** [comment]
yes but as far as I can tell not with the example provided however
```bash
 fzf --bind='alt-b:toggle+replace-query+execute(echo '{q}' >> ./somefile)+execute(cat ./somefile)'
```
works just fine so it really depends on what your goal is (based on the example given you want to print your output to the command line to use again?
